system-test/system-test-script.sh

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env bash
+# Stopping the script if any command fails
+set -e
+source bin/env.sh
+
+function cleanup {
+  echo "Removing /tmp/pulse-system-test"
+  rm  -r /tmp/pulse-system-test
+  # killing_services
+  kill_service
+  echo "Sevices are terminated/killed"
+}
+
+trap cleanup EXIT
+mkdir -p /tmp/pulse-system-test
+
+# Writing logs to local directory
+export collection_roller_log="system-test/log_files/collectionroller.log"
```

Comment:
Separate names with underscored like `collection_roller.log`. Also on the next four lines.

Reply:
fixed